packages/core/src/icons.d.ts

```diff
@@ -9,6 +9,7 @@ export type IconProps =
   | 'leader-board'
   | 'person-time-outline'
   | 'person-warning-outline'
+  | 'pix-colorful'
```

Comment:
We need to add this info in the i[con documentation](https://juntossomosmais.github.io/atomium/?path=/docs/components-icon--docs) to explain out approach about colorful icons

Reply:
My suggestion 👇 
What do you think? 

<img width="702" alt="image" src="https://github.com/user-attachments/assets/a76e9cd9-9e79-4638-ab90-a6de4271aa75">

Reply:
Perfect, we could add `` around `-colorful` to improve legibility

Reply:
https://github.com/juntossomosmais/atomium/pull/525/commits/c449af971038745261623db64f9db2fb098774eb